tchannel/testing/vcr/patch.py

```diff
@@ -83,6 +85,7 @@ def send(self, arg1, arg2, arg3,
         vcr_request = VCRProxy.Request(
             serviceName=self.service.encode('utf-8'),
             hostPort=self.hostport,
+            knownPeers=self.original_tchannel.peers.hosts,
```

Comment:
This logic patches the tornado.TChannel object. It's guaranteed to get that.

This logic will probably be changed when we invert the flow so that `request.send` is implemented on top of `call`.